lib/puppet/module/task.rb

```diff
@@ -63,11 +63,6 @@ def self.is_tasks_filename?(path)
     end
 
     def self.get_file_details(path, mod)
```

Comment:
Will this continue to work? You made a comment about `pup_module.name` not working, and that's what we use here to construct the `name`.